CMakeLists.txt

```diff
@@ -151,21 +130,28 @@ endif()
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake)
+set(LIBPMEMOBJ_CPP_REQUIRED_VERSION 1.6)
+
 find_package(PkgConfig QUIET)
 include(ExternalProject)
 include(FindThreads)
 include(CheckCXXSourceCompiles)
 include(GNUInstallDirs)
 
 include(rapidjson)
+set(PKG_REQUIRES "Requires.private: RapidJSON")
+
 if(ENGINE_VSMAP OR ENGINE_VCMAP OR ENGINE_CMAP OR ENGINE_STREE OR ENGINE_TREE3 OR ENGINE_CACHING)
 	include(libpmemobj++)
+	set(PKG_REQUIRES "${PKG_REQUIRES} libpmemobj++ >= ${LIBPMEMOBJ_CPP_REQUIRED_VERSION}")
 endif()
 if(ENGINE_VSMAP OR ENGINE_VCMAP)
 	include(memkind)
+	set(PKG_REQUIRES "${PKG_REQUIRES} memkind")
 endif()
 if(ENGINE_VCMAP OR ENGINE_CMAP)
 	include(tbb)
+	set(PKG_REQUIRES "${PKG_REQUIRES} tbb")
 endif()
 if(ENGINE_CACHING)
 	include(memcached-experimental)
```

Comment:
perhaps add here "XXX" or "TODO" for these packages. When we finally fix or replace them we could make use of pkg conf